diff --git a/inc/Utils.h b/inc/Utils.h
index d085836..de2cea2 100644
--- a/inc/Utils.h
+++ b/inc/Utils.h
@@ -68,6 +68,9 @@ public:
 	static TBool StartsWith(const TDesC8& aStr, const TDesC8& aSubstr, TBool aIgnoreCase = EFalse);
 	//static TBool EndsWith(const TDesC16& aStr, const TDesC16& aSubstr, TBool aIgnoreCase = EFalse);
 	//static TBool EndsWith(const TDesC8& aStr, const TDesC8& aSubstr, TBool aIgnoreCase = EFalse);
+	
+	// Truncate given descriptor up to aMaxLength chars and add ellipsis to the end if longer.
+//	static void /*TBool*/ TruncateLongString(TDes &aStr, TInt aMaxLength)
 	};
 
 
